http/handler.go

```
@@ -74,15 +76,26 @@ func NewHandler(db client.DB) (*Handler, error) {
 	if err != nil {
 		return nil, err
 	}
+	// generate a secure random audience value
+	audienceBytes, err := crypto.RandomBytes(64)
+	if err != nil {
+		return nil, err
+	}
+	audience := hex.EncodeToString(audienceBytes)
 	txs := &sync.Map{}
 
 	mux := chi.NewMux()
 	mux.Route("/api/"+Version, func(r chi.Router) {
 		r.Use(
 			ApiMiddleware(db, txs),
 			TransactionMiddleware,
-			IdentityMiddleware,
+			AuthMiddleware(audience),
 		)
+		r.Get("/audience", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Add("Content-Type", "text/plain")
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(audience)) //nolint:errcheck
```

todo: Please document why it is safe to ignore this error, if it is safe.